packages/x-components/tests/unit/main-scroll.spec.ts

```diff
@@ -141,18 +139,17 @@ describe('testing MainScroll component', () => {
         userScrolledToElementSpy().should('have.been.calledWith', 'item-5');
       });
 
-      it('ignores the first pixels of scroll', () => {
+      it.only('ignores the first element of scroll', () => {
         const { scrollToItem, userScrolledToElementSpy } = renderMainScroll({
           ...defaultParameters,
-          minScrollTrigger: 75,
           itemHeight: '50px'
```

Comment:
You're using the same value as the default, so please remove it or change the value to express why are you sending it to the render function.

Reply:
I'm going to remove it because this is no longer relevant for the test. The test does no longer depend on element heights. 
With the previous approach it was dependant, so I would rather be explicit about my test inputs and repeat them than let them be implicit and confusing.